src/apibuilder-cli/file_tracker.rb

```diff
@@ -25,6 +25,11 @@ def initialize(project_dir, opts={})
                       end
         end
       end
+      @subset = !!opts.delete(:subset)
+      if @subset
+        @current = @previous
```

Comment:
I think you should be filtering here (and for `@current_raw` below) based on `org` and `app` (this would imply passing those actual values into the constructor, and not just a boolean). The logic you've added here simply copies over all `.tracked_files` indiscriminately. The edge case you're not covering (at least as far as I can tell) is when a file is removed from the generated code. All this previous/current business is there to enable us to know which files were there previously and delete them if the generator stops producing them. My read of your changes is that all files in the subset will continue to be tracked, even when removed from the generator.

It also seems likely (based on the code here: https://github.com/apicollective/apibuilder-cli/pull/60/files#diff-790cbc02d9b72fde257906d9fbf9ee66L70) that you'd want to remove from `@previous` too, when selecting into `@current`.